projects/ad9081_fmca_ebz/a10soc/system_constr.sdc

```diff
@@ -0,0 +1,12 @@
+
+create_clock -period "10.000 ns"  -name sys_clk_100mhz      [get_ports {sys_clk}]
+create_clock -period  "4.000 ns"  -name ref_clk             [get_ports {fpga_refclk_in}]
+create_clock -period  "6.000 ns"  -name device_clk          [get_ports {clkin6}]
```

Comment:
For the default profile, shouldn't this be constrained at 4ns ?

Reply:
V1.  
- rebased to master
- updated constraints for device clock

Reply:
For me, the new design fails timing